### PR TITLE
fix device mapper data collecting cycle calculation error

### PR DESCRIPTION
**Pull request by tangming1996**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
The device mapper data collecting cycle is too short, causing the mapper to crash.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by wbc6080 — COMMENTED** [review]
Users can set the frequency for mapper to report data in the device-instance configuration file refer to [page](https://deploy-preview-439--competent-hypatia-4195b7.netlify.app/docs/concept/device/device_crds#device-instance-sample). The default is to report once per second.

**Comment by wbc6080** [comment]
cycle

**Comment by tangxiaoming1** [comment]
> cycle

Thank you for your reply. I understand what you mean, but the unit of data collecting cycle is defined by mapper. I think millisecond as the minimum unit is sufficient, and the readability will be higher. I used 1000 nanoseconds in the real environment, and mapper just crashed.

**Review by Shelley-BaoYue — COMMENTED** [review]
There are something wrong with ci and I have solved it and merge it into master. Please rebase the new master branch and push again. Thx : ）

**Comment by tangming1996** [comment]
> There are something wrong with ci and I have solved it and merge it into master. Please rebase the new master branch and push again. Thx : ）

Okay, I already rebase master.

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

File: staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ func dataHandler(ctx context.Context, dev *driver.CustomizedDev) {
 			ObservedDesired: twin.ObservedDesired,
 			VisitorConfig:   &visitorConfig,
 			Topic:           fmt.Sprintf(common.TopicTwinUpdate, dev.Instance.ID),
-			CollectCycle:    time.Duration(twin.Property.CollectCycle),
+			CollectCycle:    time.Millisecond * time.Duration(twin.Property.CollectCycle),
```
Comment (fisherxu) [inline staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go:134]: Thanks for the fixing, and the unit here is ms or second? Any comments or docs?

Comment (tangming1996) [reply staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go:134]: Thank you very much for pointing out the problem. I have added the explanation in the document and also modified the time unit of the reporting cycle.

File: staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/influxdb2/handler.go [hunk 2]
```diff
@@ -20,8 +20,6 @@ import (
 	"context"
 	"time"
 
-	"k8s.io/klog/v2"
```
Comment (wbc6080) [inline staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/influxdb2/handler.go:23]: Why remove this dependence?

File: staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/influxdb2/handler.go [hunk 3]
```diff
@@ -37,7 +35,7 @@ func DataHandler(ctx context.Context, twin *common.Twin, client *driver.Customiz
 		klog.Errorf("init database client err: %v", err)
 		return
 	}
-	reportCycle := time.Duration(twin.Property.ReportCycle)
+	reportCycle := time.Millisecond * time.Duration(twin.Property.ReportCycle)
```
Comment (wbc6080) [inline staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/influxdb2/handler.go:38]: Other database still need to be fixed like tdengine

File: staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go (hunk 1 above)
Comment (wbc6080) [inline staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go:134]: The reportcycle in this file need to be fixed (like in **pushHandler** function)

**Review by wbc6080 — COMMENTED** [review]
Pleause pass the CI test, you can re-run them by reopen this PR

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**